charts/placement-policy-scheduler-plugins/templates/rbac.yaml

```diff
@@ -1,8 +1,24 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: pp-plugins-scheduler
+  name: system:aggregated-pp-plugins-scheduler-reader
```

Comment:
Why do we need the `system` prefix for the placement policy cluster role?

Reply:
No need for it here, removed.